src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/util/LicenceEventType.kt

```diff
@@ -7,6 +7,8 @@ enum class LicenceEventType {
   APPROVED,
   ACTIVATED,
   SUPERSEDED,
+  HARDSTOP_CREATED,
+  HARDSTOP_SUBMITTED,
```

Comment:
I am thinking aloud, if we need new types or existing types can suffice. In my head HardStop is still a licence and statuses are similar, its just being created/triggered by a different place. not sure. I don't see any real issues with two new status either though.